crates/test-modules/modules/crates/resource-main/src/main.rs

```diff
@@ -0,0 +1,27 @@
+witx_bindgen_rust::import!("../../../tests/resource.witx");
```

Comment:
FWIW I switched these to using custom `*.witx` files inside each crate since the top-level `tests` folder is currently used by the code generators for "can I generate valid code for this interface". I should probably rename that folder at some point now that we've merged wasmlink though... In any case having to keep all this in sync can be sort of difficult, so I think it may be best to have this be a local witx file for just this crate?

Reply:
That works for me, I'll copy each witx local to the exporting test module.